libs/component-library/src/lib/breadcrumbs/breadcrumbs.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { FxBox, FxBoxProps } from '../box/box';
+import { FxChevronRightIcon } from '../icons/icons';
+import { FxPressableOpacity } from '../pressable-opacity/pressableOpacity';
+import { FxSpacer } from '../spacer/spacer';
+import { FxText, FxTextProps } from '../text/text';
+import { useFxTheme } from '../theme/useFxTheme';
+
+type PathBase = {
+  label: FxTextProps['children'];
+  onPress: (item: PathBase) => void;
+};
+
+export type FxBreadcrumbsProps = FxBoxProps & {
+  path: PathBase[];
+};
+
+export const FxBreadcrumbs = ({ path, ...rest }: FxBreadcrumbsProps) => {
+  const { colors } = useFxTheme();
+
+  return (
+    <FxBox flexDirection="row" alignItems="center" {...rest}>
+      {path.map((item, idx) => (
+        <>
+          <FxPressableOpacity
+            onPress={() => item.onPress(item)}
+            alignItems="center"
+            justifyContent="center"
+          >
+            <FxText variant="bodyXSRegular" color="content1">
```

Comment:
I think the last breadcrumb in the list is supposed to have the `bodyXSSemibold` text style

Reply:
AH great catch @keithluchtel .. that attention to detail is on point!

Reply:
Fixed